src/service/video-manager-impl.js

```diff
@@ -405,6 +478,30 @@ class VideoEntry {
       mutate,
     });
   }
+
+
+  /**
+   * Returns whether the video is paused or playing after the user interacted
+   * with it or playing through autoplay
+   * @return {!../video-interface.VideoInterface} PlayingStates
+   */
+  getPlayingState() {
+    if (this.userInteracted_ && this.isPlaying_) {
```

Comment:
we need an extra variable to track autoplay. autoplay is not guaranteed, many platforms don't support it so just having `autoplay` attribute does not mean video will autoplay. 

set something along the lines of `this.playCalledByAutoplay_` in `autoplayLoadedVideoVisibilityChanged_()` where it calls `this.video.play(/*autoplay*/ true);`, then this logic will become:

```
if (!this.isPlaying_) {
  return PlayingStates.PAUSED;
}

if (this.isPlaying_ && this.playCalledByAutoplay_ && !this.userInteracted_) {
   return PlayingStates.PLAYING_AUTO;
}

return PlayingStates.PLAYING_MANUAL; 
```